os/include/tinyara/log_dump/log_dump.h

```diff
@@ -26,13 +26,23 @@
  * Definitions
  ********************************************************************************/
 
- #define LOGDUMP_SAVE_START	"1"
- #define LOGDUMP_SAVE_STOP	"2"
+/* Log Dump Thread information */
+#define LOG_DUMP_NAME        "log_dump"		/* Log dump thread name */
+#define LOG_DUMP_STACKSIZE   16384		/* Log dump thread stack size */
+#define LOG_DUMP_PRIORITY    100		/* Log dump thread priority */
+
+#define LOGDUMP_SAVE_START	"1"
+#define LOGDUMP_SAVE_STOP	"2"
 
 /********************************************************************************
  * Public Types
  ********************************************************************************/
 
+struct log_dump_msg_s {
+      char ch;
+      bool comp;
+};
```

Comment:
This is used only in the log_dump.c. Could you check this is necessary here?

Reply:
moved to log_dump.c